lib/src/over_react_redux/over_react_flux.dart

```diff
@@ -28,9 +28,9 @@ class FluxToReduxAdapterStore<S extends flux.Store> extends redux.Store<S> {
 
   StreamSubscription _storeListener;
 
-  FluxToReduxAdapterStore(this.store, dynamic actions) : super(_noopReducer, initialState: store, distinct: false) {
+  FluxToReduxAdapterStore(this.store, dynamic actions, {Function reducer}) : super(reducer ?? _noopReducer, initialState: store, distinct: false) {
```

Comment:
We should add a middleware option, as one of the biggest benefits of moving to redux allowing them to use the `overReactReduxDevToolsMiddleware` while transitioning seems like an added benefit, also, we should type `reducer` a bit more strictly than just `Function`
```suggestion
  FluxToReduxAdapterStore(this.store, dynamic actions, {{List<redux.Middleware<S>>middleware, redux.Reducer<S> reducer}) : super(reducer ?? _noopReducer, initialState: store, middleware: middleware, distinct: false) {
```

Reply:
Good catch!!

Reply:
forgot to add ` = const []` woops should be `List<redux.Middleware<S>>middleware = const []`

Reply:
Hahaha don't worry, I ran into it and it was pretty easy to see what was going on